assets/blocks/quiz/quiz-block/quiz-edit.js

```diff
@@ -44,12 +44,20 @@ const QuizEdit = ( props ) => {
 		setExistingQuestionsModalOpen,
 	] = useState( false );
 
-	const openExistingQuestionsModal = () =>
-		setExistingQuestionsModalOpen( true );
-
 	const closeExistingQuestionsModal = () =>
 		setExistingQuestionsModalOpen( false );
 
+	/* Temporary solution. See https://github.com/WordPress/gutenberg/pull/29911 */
+	const renderAppenderComponent = useCallback(
```

Comment:
I think it's fine as a "final" solution too. For the naming, isn't `renderAppender` enough?

Reply:
Thank you for the review!

I don't think it's a good solution because we're creating a component function in the render. 🤔
About the name, it's to make it more transparent that it's a component and not a render prop function, to not create confusion while reading the code. Maybe `RenderAppenderComponent` could be a better name. But I'm merging it for now and anything we can change in the future. :)

Reply:
After merged it's bothering me. haha. I'll create a quick PR to rename it to `AppenderComponent`.

Reply:
Nah it's fine :)